homeassistant/util/logging.py

```diff
@@ -83,20 +95,10 @@ def async_activate_log_queue_handler(hass: HomeAssistant) -> None:
         migrated_handlers.append(handler)
 
     listener = logging.handlers.QueueListener(simple_queue, *migrated_handlers)
+    queue_handler.listener = listener
 
     listener.start()
 
-    @callback
-    def _async_stop_queue_handler(_: Any) -> None:
-        """Cleanup handler."""
-        # Ensure any messages that happen after close still get logged
-        for original_handler in migrated_handlers:
-            logging.root.addHandler(original_handler)
-        logging.root.removeHandler(queue_handler)
```

Comment:
I don't think this is needed anymore, since we now stop the queue listener when its `close` method is called?

Reply:
`close` is called from `logging`'s `atexit` handler, I don't think it's possible for any logging to happen after that?